ngraph/core/include/ngraph/op/space_to_depth.hpp

```diff
@@ -70,6 +71,9 @@ namespace ngraph
                 virtual std::shared_ptr<Node>
                     clone_with_new_inputs(const OutputVector& new_args) const override;
 
+                bool evaluate(const HostTensorVector& outputs,
```

Comment:
Please inherit from simple op, not fused op and remove decomposition completely

Reply:
Fixed